deployer/deploy_actions.py

```diff
@@ -245,9 +245,8 @@ def generate_helm_upgrade_jobs(changed_filepaths):
     ) = discover_modified_common_files(changed_filepaths)
 
     # Convert changed filepaths into absolute Posix Paths
-    changed_filepaths = [
-        Path(os.getcwd()).joinpath(filepath) for filepath in changed_filepaths
-    ]
+    root_dir = Path(__file__).parent.parent
+    changed_filepaths = [root_dir.joinpath(filepath) for filepath in changed_filepaths]
 
     # Get a list of filepaths to all cluster.yaml files in the repo
     cluster_files = get_all_cluster_yaml_files()
```

Comment:
@GeorgianaElena this is in the codebase, but `set()` is not being called on the result 🤔